pkg/apis/build/v1alpha1/buildpack_metadata.go

```diff
@@ -2,14 +2,15 @@ package v1alpha1
 
 type BuildpackMetadataList []BuildpackMetadata
 
+// +k8s:openapi-gen=true
 type BuildpackMetadata struct {
-	ID      string `json:"key"`
+	Key     string `json:"key"`
```

Comment:
Whoops, should that be json id?

Reply:
The generator complains if the struct member and the json name are not the same. This has always been serialized as key. I'd like to change it to Id but I think thats for another commit.

Reply:
Fair enough. Is it easier to rename the key than change the json?

Reply:
I deferred to changing the struct because I'm not sure how the data is consumed after it is serialized. Changing the json could have larger consequences.